src/reducers/index.js

```diff
@@ -24,8 +23,8 @@ export const getConnections = state =>
     )
     .map(connection => ({
       account: fromAccounts.getAccount(state.cozy, connection.accountId),
-      konnector: fromKonnectors.getKonnector(
-        state.cozy,
```

Comment:
what was state.cozy ? the triggers ?

Reply:
state.cozy is the state's property used by CozyClient to store everything which is received by the stack.